### ISSUE TITLE
Add source-of example to techniques?

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#1600]
The use of `refines` was expected to be deprecated when we created the accessibility specification so we don't have an example of using the `source-of` property (we recommended the [property prior to 3.1](http://idpf.org/epub/301/spec/epub-publications.html#sec-opf-dcsource)).

Should we at least demonstrate the property again in the techniques document?

**Comment by avneeshsingh** [comment]
Currently epub-type does not has any effect on accessibility. Screen readers completely ignore it and I do not think that operating systems accessibility API recognizes it either. If some reading systems are using it for improving accessibility, I would be glad to know about it.

Taking the overall view of current implementations, it does not look that this attribute directly help in accessibility. On the other hand DPUB  aria roles are exposed by browsers in their accessibility tree. Screen readers do not support DPUB aria also, but at least this information is exposed by user agents. 

Therefore I am concerned that mentioning epub-type in EPUB accessibility techniques will create a wrong perception that epub-type is important for accessibility.
We can mention that originally epub-type was provided for accessibility, but then it was superseded by DPUB aria due to better compatibility with existing accessibility frameworks. (it is quite possible that it is mentioned somewhere already)

**Comment by mattgarrish** [comment]
Sorry, I obviously wasn't thinking as I was typing this issue out.

What I meant to write is that the `refines` attribute was meant to be deprecated.

The issue here isn't about `epub:type` and its lack of implementation, but whether we should be identifying the `dc:source` element that is the source of the pagination using a `source-of` property in the package metadata.

This is about being able to clearly identify the source in the package document so that reading systems can use it or vendors can find it for display. It's not about exposing this information directly for AT to consume.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-04-08-epub-a11y#section1) on 2021-04-08

List of resolutions:

- ***[Resolution No. 1](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-04-08-epub-a11y#resolution1):  Add accessibility technique for `dc:source` metadata.***

<details><summary><i>View the transcript</i></summary>

### 1. Source-of example to techniques document 

_See github issue [#1600](https://github.com/w3c/epub-specs/issues/1600)._




**Matt Garrish:** just a question going back to 3.1 when we were expecting to drop refines  
… up to 3.1 we used to say it was required to put source-of when you had pagelist  
… that is gone now  
… do we still want to recommend source-of?  
… given that there can be multiple `dc:source` elements in an epub  
… the example would mostly be for guidance, an example  

**Avneesh Singh:** so it would not be normative, it would be a technique  

**Matt Garrish:** yes, just a demonstration  

**Gregorio Pellegrino:** i think this requirement is checked by epubcheck or ACE  

**Matt Garrish:** yes, it is required to have `dc:source`, but not required to have that source identified by this attribute  

**Avneesh Singh:** so, should we include an example of this in the techniques?  
… i think there is no harm in it  

**Charles LaPierre:** in GCA we just got a book from WIPO, and they have a `dc:source`, but they had a refines there pointing to ONIX codelist 5  
… instead of pointing to something about pagination  

**Gregorio Pellegrino:** that means that the source code is an ISBN  

**Charles LaPierre:** we've been recommending the use of source-of in GCA  

> **Proposed resolution: Add accessibility technique for `dc:source` metadata.** *(Avneesh Singh)*

> *Gregorio Pellegrino:* +1

> *Matthew Chan:* +1

> *Charles LaPierre:* +1

> *Wendy Reid:* +1

> *Matt Garrish:* +1

> *Ben Schroeter:* +1

> *George Kerscher:* +1

> ***Resolution #1: Add accessibility technique for `dc:source` metadata.***


</details>
